doc/examples/features_detection/plot_glcm.py

```diff
@@ -36,14 +36,14 @@
 image = data.camera()
 
 # select some patches from grassy areas of the image
-grass_locations = [(474, 291), (440, 433), (466, 18), (462, 236)]
+grass_locations = [(153, 200), (175, 225), (200, 250), (175, 300)]
 grass_patches = []
 for loc in grass_locations:
     grass_patches.append(image[loc[0]:loc[0] + PATCH_SIZE,
                                loc[1]:loc[1] + PATCH_SIZE])
 
 # select some patches from sky areas of the image
-sky_locations = [(54, 48), (21, 233), (90, 380), (195, 330)]
+sky_locations = [(25, 25), (50, 200), (75, 250), (100, 275)]
```

Comment:
Last Q: are these correct now?

Other than that, this is ready, if we can get someone else to agree on what to do about the failing rank filter tests. Thank you @alexdesiqueira!!!